### PR TITLE
Feat: Changed first heading of the page 

### LINKED ISSUE #1: Feedback: Change the heading of the last line
the heading of the last line is very silly

### PR DESCRIPTION
**Pull request by SumitKumar777**
i have changed the heading in the new branch made out of master and pushed the code and right  now im creating a pull request to the ashu 
fixes #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <script src="script.js"></script>
 </head>
+
 <body>
-    <h1>this is my first line of code </h1>
+    <h1>this is the new heading </h1>
```
Comment (Aashu-toshh) [inline index.html:12]: Thanks for the this change this was very helpfull

Comment (SumitKumar777) [reply index.html:12]: thanks ashu for this review

Comment (Aashu-toshh) [reply index.html:12]: Can you do more contribution to this repo

Comment (SumitKumar777) [reply index.html:12]: yeah offcourse im going to do this for me

**Pull request merged**